src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -40,6 +40,13 @@ module sonic-device_metadata {
                     type stypes:hwsku;
                 }
 
+                leaf asic_id {
+                    type string {
+                        length 0..16;
```

Comment:
>0 [](http://example.com/codeflow?start=31&length=1)

Is 0 valid?

Reply:
Yes, it is.

https://www.rfc-editor.org/rfc/rfc7950#section-9.4.4: 
> A length value is a non-negative integer or one
   of the special values "min" or "max".